### DISCUSSION TITLE
Get list of blocked cookies on initialization

### DISCUSSION THREAD

**Discussion by Akikyo** [orestbida/cookieconsent#679] [Q&A]
We have plenty of sites running this cookie consent and one thing is we would like to automate it more. The idea is to get collection of blocked cookies on initialization (before any action). So i could sort out them into proper categories, generate list of them for GUI instead setting it manually in config. Is there way to get it from cookiesconsent or else?

**Comment by orestbida** [comment]
This is not possible via the plugin; you'd need a dedicated service that analyzes http request and the content of the website to determine what cookies are in use.

I'm not aware of any service that returns a nice response with the list of cookies.
